containers/src/Data/IntMap/Internal.hs

```diff
@@ -2578,6 +2579,14 @@ filter :: (a -> Bool) -> IntMap a -> IntMap a
 filter p m
   = filterWithKey (\_ x -> p x) m
 
+-- | \(O(n)\). Filter all keys that satisfy some predicate.
+--
+-- > filterKeys (> 4) (fromList [(5,"a"), (3,"b")]) == singleton 5 "a"
+-- > filterKeys (> 4) == filterWithKey (\k _ -> k > 4)
```

Comment:
I had put that there as example for the documentation for the users. Shall i not just keep that? I'm not familiar with `@` in a docblock, is it a test? I can add those tests and at the same time keep `prop_filterKeys` from `intmap-properties` and `map-properties`

Reply:
No it's not a test, just plain documentation. It will look like this: https://hackage.haskell.org/package/containers-0.7/docs/Data-Map-Lazy.html#v:take

Anyway, you've added it below so please remove this line "`> filterKeys (> 4) == filterWithKey (\k _ -> k > 4)`".